BubblesKotlin/app/src/main/java/com/example/android/bubbles/data/NotificationHelper.kt

```diff
@@ -117,6 +152,7 @@ class NotificationHelper(private val context: Context) {
             .setContentTitle(chat.contact.name)
             .setSmallIcon(R.drawable.ic_message)
             .setCategory(Notification.CATEGORY_MESSAGE)
+            .setShortcutId(chat.contact.id.toString())
```

Comment:
Nit: I'd use a longer id here. Maybe: `"contact_${chat.contact.id}"`?

Reply:
Done. Added a val in Contact.